src/api/common/utils.js

```diff
@@ -20,8 +20,20 @@ function convertAmount(amount) {
   };
 }
 
+function wrapCatch(asyncFunction: any): any {
```

Comment:
Interesting. Well it doesn't fully constrain the type, but I guess it's would at least check that the parameter is a function with no return value, so I'll make the change.